recipes/eastl/all/conanfile.py

```diff
@@ -95,4 +95,4 @@ def package_info(self):
         if self.settings.os == "Linux":
             self.cpp_info.system_libs.append("pthread")
         if self.options.shared:
-            self.cpp_info.defines.append("EASTL_DLL")
+            self.cpp_info.defines.append("EA_DLL")
```

Comment:
Indeed it looks better: https://github.com/electronicarts/EASTL/blob/b2d923a7af2751e4e41036093c3f839524f01a49/include/EASTL/internal/config.h#L244